### PR TITLE
Add workaround to switch validation logic regarding E-switches.

### PR DESCRIPTION
**Pull request by dpoltavets**
Closes: #2562.

### CODE REVIEW

File: services/wfm/src/test/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImplTest.java [hunk 1]
```diff
@@ -210,11 +245,28 @@ private PersistenceManager build() {
...
 
-            Flow flow = mock(Flow.class);
-            when(flow.getFlowId()).thenReturn("test_flow");
-            when(flowPathA.getFlow()).thenReturn(flow);
+            Flow flowA = mock(Flow.class);
+            when(flowA.getFlowId()).thenReturn("test_flow");
+            when(flowPathA.getFlow()).thenReturn(flowA);
+
+            Switch switchE = Switch.builder()
+                    .switchId(SWITCH_ID_B)
```
Comment (rtretyak) [inline services/wfm/src/test/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImplTest.java]: `SWITCH_ID_E` ?

Comment (rtretyak) [reply services/wfm/src/test/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImplTest.java]: Now I'm not sure how these tests pass considering the ID is wrong

Comment (dpoltavets) [reply services/wfm/src/test/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImplTest.java]: We don't get switch by ID. And when we call `flowPathB.getSrcSwitch()`, we get switchE with "E" manufacturer (when(flowPathB.getSrcSwitch()).thenReturn(switchE)). But I've fixed this mistake.

File: services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java [hunk 2]
```diff
@@ -216,7 +225,17 @@ private MeterInfoEntry makeMisconfiguredMeterEntry(FlowPath path, MeterEntry met
                 .build();
     }
 
-    private boolean equalsBurstSize(long actual, long expected) {
-        return Math.abs(actual - expected) <= METER_BURST_SIZE_EQUALS_EPS;
+    private boolean equalsRate(long actual, long expected, boolean isESwitch) {
+        if (isESwitch) {
```
Comment (timofei-durakov) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java]: Please add note here

File: services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java [hunk 3]
```diff
@@ -216,7 +225,17 @@ private MeterInfoEntry makeMisconfiguredMeterEntry(FlowPath path, MeterEntry met
...
+            return Math.abs(actual - expected) <= expected * E_SWITCH_METER_RATE_EQUALS_DELTA_COEFFICIENT;
+        }
+        return actual == expected;
+    }
+
+    private boolean equalsBurstSize(long actual, long expected, boolean isESwitch) {
+        if (isESwitch) {
```
Comment (timofei-durakov) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java]: and here

File: services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java [hunk 4]
```diff
@@ -42,7 +43,9 @@
 
 @Slf4j
 public class ValidationServiceImpl implements ValidationService {
-    private static final int METER_BURST_SIZE_EQUALS_EPS = 1;
+    private static final int METER_BURST_SIZE_EQUALS_DELTA = 1;
```
Comment (timofei-durakov) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java]: I would prefer to see this configurable

Comment (timofei-durakov) [reply services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java]: But could be done in separate patch

File: services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java (hunk 2 above)
Comment (dpoltavets) [reply services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java]: Added.

File: services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java (hunk 3 above)
Comment (dpoltavets) [reply services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImpl.java]: Added.

**Pull request merged**